security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -237,13 +235,10 @@ func toEnvoySecret(s *security.SecretItem) *tls.Secret {
 	return secret
 }
 
-func pushLog(names []string, err error) {
-	if err != nil {
-		return
-	}
+func pushLog(names []string) model.XdsLogDetails {
 	if len(names) == 1 {
-		sdsServiceLog.WithLabels("resource", names[0]).Infof("SDS: PUSH")
-	} else {
-		sdsServiceLog.WithLabels("resources", len(names)).Infof("SDS: PUSH")
+		// For common case of single resource, show which resource it was
```

Comment:
For the case of multiple resources, the code in the PR returns an empty XdsLogDetails{}, whereas the original code returns a log of the number of resources. Is the case of multiple resources an erroneous or exceptional case? If so, a log for such case may be helpful to detect the problem.

Reply:
@lei-tang the core ads code (in pilot's ads.go) is always logging. XdsLogDetails just allows adding *additional* info to log as well.